library/include/G4CMPSolidUtils.hh

```diff
@@ -0,0 +1,59 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file library/include/G4CMPSolidUtils.hh
+/// \brief Definition of the G4CMPSolidUtils class
```

Comment:
Ideally, you would describe what G4CMPSolidUtils is used for, or how it works, in a couple of sentences.  This line is meant as the "template placeholder", but I have been really horrible about doing this properly myself.